.ci/update-github.sh

```diff
@@ -24,7 +24,14 @@ if [ "$TARGET_VERSION" != "$CURRENT_VERSION" ]; then
   exit 1;
 fi
 
-./.ci/update-github-milestones.sh "$TARGET_VERSION"
+echo "Go to folder where site was build and sources are already at required tag"
+cd target/checkout
```

Comment:
there is not such folder.
do checkout by tag `git checkout checkstyle-$TARGET_VERSION`

Reply:
I am doing the changes but I am not sure if we have a tag ( for eg: the current version, `10.3.4` ) but at this moment we have `10.3.3` which is latest tag

Reply:
Release:prepare will create tag 10.3.4.
This action will execute a bit later , it should expect that tag exists.
This action should be tested on 10.3.3 (any previous tag). Imagine that release (tag creation and copy of jars) was done and now we catch-up on updates of items in GitHub to beautify GitHub content.